.npmrc

```diff
@@ -1,2 +0,0 @@
-@fortawesome:registry=https://npm.fontawesome.com/
-//npm.fontawesome.com/:_authToken=${FONTAWESOME_NPM_AUTH_TOKEN}
```

Comment:
It's because your PR delete the `.npmrc` file. Was it intentional?

Reply:
Not that I recall

Reply:
The commit from the linter action did
https://github.com/gemini-hlsw/explore/pull/1502/commits/a8607aaffd70738e3f085dc44f24a5bca71525aa

Reply:
Yes I just realize that too. Huh.

Reply:
Oh, I know the problem. It's because the `npm install` step has a `rm .npmrc` sub-step in there. I copied it from the old build but I wasn't sure why you need it.